server/http/tests/server.rs

```diff
@@ -107,7 +107,7 @@ mod test {
                         // Send an empty JSON POST request
                         let client = Client::new();
                         let request =
-                            Request::post(format!("http://localhost:8001/subgraphs/id/{}", id))
+                            Request::post(format!("http://localhost:8007/subgraphs/id/{}", id))
```

Comment:
This is unrelated, just picked a port that would stop conflicting with graph-node defaults.